.gitignore

```diff
@@ -202,6 +202,7 @@ app-keypair.json
 
 # CI
 combined-patch-file.txt
+.circleci/continue_config.yml
```

Comment:
What's this for?

Reply:
We added `npm run validate-ci-config` which generates this file